apps/frontend/src/components/LoginPage.tsx

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import {
+  Box,
+  Button,
+  Flex,
+  FormControl,
+  FormLabel,
+  Input,
+  Stack,
+  Heading,
+  VStack,
+} from "@chakra-ui/react";
+import { HamburgerIcon, ChevronLeftIcon } from "@chakra-ui/icons";
+
+function LoginPage() {
+  return (
+    <>
+      <Box w={"100vw"} h={"100vh"} bg={"white"}>
```

Comment:
You can use ```Header``` component once you rebase here

Reply:
Made a comment in my code to do just this. Avoiding rebasing for now, since, according to Ben, there are some conflicts we have to fix.